### PR TITLE
[bitnami/mongodb-sharded] Fix helm error in arbiter statefulset

### PR DESCRIPTION
**Pull request by lucchmielowski**
**Description of the change**

When doing a `helm template -f my_values_file.yaml` I had the following issue : 

```
Error: template: mongodb-sharded/templates/shard/shard-arbiter-statefulset.yaml:268:28: executing "mongodb-sharded/templates/shard/shard-arbiter-statefulset.yaml" at <.Values.diagnosticMode.enabled>: can't evaluate field Values in type int
```

Looks like we may have missed the necessary `$` in front of the `Values` when adding the diagnostic mode (https://github.com/bitnami/charts/pull/7012)

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - Looks like no issue was opened 🤷 

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/mongodb-sharded/Chart.lock [hunk 1]
```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: common
   repository: https://charts.bitnami.com/bitnami
-  version: 1.7.0
-digest: sha256:6786bc7fdfc6e4038fd28819c2b07339f232282dbaeab21de8064aaa6814a8d1
-generated: "2021-07-07T15:32:21.159182482Z"
+  version: 1.7.1
+digest: sha256:d05549cd2eb5b99a49655221b8efd09927cc48daca3fa9f19af0257a11e5260f
```
Comment (lucchmielowski) [inline bitnami/mongodb-sharded/Chart.lock:5]: This changed when installing the required other packages, not sure if I should keep it or not 🤷

Comment (carrodher) [reply bitnami/mongodb-sharded/Chart.lock:5]: We can remove this file from the PR, no problem

**Review by carrodher — APPROVED** [review]
Thanks for your changes! LGTM

**Comment by lucchmielowski** [comment]
Thanks @carrodher! looks like I missed some `.Values` though 🙈

**Comment by lucchmielowski** [comment]
> Thanks @carrodher! looks like I missed some .Values though 🙈

NVM, realized that the missing ones are using the `context()` with the `$` as a parameter 👍

**Comment by lucchmielowski** [comment]
Also, should I open an issue about improving the CI for this chart ? (I believe we did not see the issue because of the `.Values.shardsvr.arbiter.replicas` being set to `0` by default 🤔)

**Pull request merged**